common/match/pool.ts

```diff
@@ -253,7 +253,7 @@ export const TEST_POOL = {
 const _pools = [
     {
         name: 'lern-fair-now',
-        confirmInterest: false,
+        confirmInterest: true,
```

Comment:
This needs to stay false

Reply:
Then I misunderstood our conversation 😅 
I'll revert the change.

Reply:
Yeah we do not want to invite new users for interest confirmation, which this toggle controls